lib/Jobs/PasswordExpirationNotifierJob.php

```diff
@@ -192,7 +192,7 @@ private function sendPassExpiredNotification(OldPassword $passInfo, $expirationT
 		$notification->setApp('password_policy')
 			->setUser($passInfo->getUid())
 			->setDateTime(new \DateTime("@{$notificationTimestamp}"))
-			->setObject('expired', $passInfo->getId())
+			->setObject('expired', (string)$passInfo->getId())
```

Comment:
Please share the error which is fixed through this

Reply:
``$passInfo->getId()`` returns an ``int`` but ``setObject()`` expects a string.